source/index.js

```diff
@@ -272,8 +269,25 @@ app.get('/sync-user', (req, res) => {
   }
 });
 
+if (CONFIG.NODE_ENV === 'development') {
+  // allows service worker to run
+  // In prod, this is handled by nginx
+  app.use((req, res, next) => {
+    res.setHeader('Service-Worker-Allowed', CONFIG.ROOT_URI || '/');
```

Comment:
Great! good catch! Didn't note the header path was not updated! Hope this fixes the push feature